flight/mcu/lib/teensyShared/ActiveControl.cpp

```diff
@@ -82,14 +93,29 @@ bool ActiveControl::ActiveControl_ON() {
             active_control_on = false;
             break;
         case STATE_COAST:
-            active_control_on = true;
+            // This adds a delay to the coast state so that we don't deploy flaps too quickly
+            if (*ac_coast_timer > coast_ac_delay_thresh) {
```

Comment:
this ac_coast_timer is never set, de referencing it is a nullptr de reference

Reply:
@gautamdayal this is a you think I think?

Reply:
Fixed. Since I named another variable the exact same way as the member variable, it still worked lmao, but I made it use the member variable now.